ERCS/erc-4527.md

```diff
@@ -135,8 +135,8 @@ To send the unsigned data from a watch-only wallet to an offline signer, the new
 
 #### CDDL for Eth Sign Request.
 
-The following specification is written in Concise Data Definition Language `CDDL`.
-UUIDs in this specification notated UUID are CBOR binary strings tagged with #6.37, per the IANA `CBOR Tags Registry`.
+The following specification is written in Concise Data Definition Language (CDDL).
+UUIDs in this specification notated UUID are CBOR binary strings tagged with #6.37, per the IANA CBOR Tags Registry.
```

Comment:
Same kind of external link problem here. The IANA CBOR Tags Registry isn't a permitted external resource. I think you could get away here just removing this text.

Reply:
updated